src/pynwb/ndx_hierarchical_behavioral_data/transcription_io.py

```diff
@@ -0,0 +1,64 @@
+
+import os
+import glob
+from ndx_hierarchical_behavioral_data.definitions.transcription import phonemes, syllables, words, sentences
+
+
+def read_transcription_data(path_to_files, filename_pattern):

```

Comment:
pandas.read_csv might be better here.

Reply:
Data files are not in .csv format and also do not have headings.

Reply:
```python
import pandas as pd
pd.read_csv('mbbr0_si2315.phn',
            names=['start_time', 'stop_time', 'label'],
            sep=' ')
```